src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/lsf/rest/LSFRestJobManager.groovy

```diff
@@ -61,7 +62,11 @@ class LSFRestJobManager extends AbstractLSFJobManager {
         this.restExecutionService = restExecutionService as RestExecutionService
     }
 
-    /**
+    @Override
+    boolean checkCompatibilityToHost() {
```

Comment:
Why not check here that the REST endpoint indeed speaks LSF REST? With a not-implemented exception we will certainly need special try-catch in client code.